config/locales/en/loan_questions.en.yml

```diff
@@ -17,6 +17,18 @@ en:
     required_loan_types: "Required for which loan types?"
     text: "Title"
 
+  linked_document:
+    cell_end: End Cell
+    cell_end_hint: Z26
+    cell_start: Start Cell
+    cell_start_hint: A1
+    delete: Delete Document
```

Comment:
@rgalusca , this is the line you would need to change. The text is only used once as the text for the delete link.